eventActions/highlightActions.js

```diff
@@ -1,11 +1,13 @@
 const Highlights = require('../databaseFiles/highlightsTable.js');
 const Discord = require('discord.js');
+const config = require('../config.json');
 
 class highlightActions {
 
 	// Method to call to check a message for a highlighted message
 	static async checkForHighlight(client, message){
 		if(message.content.charAt(0) === '!') return; // Ensure commands aren't caught
```

Comment:
This should use the prefix from the config file

Reply:
I'm confused on what this means 😕 like making sure it uses the prefix value? Isn't it default used for any commands?